cms/articles/tests/test_models.py

```diff
@@ -251,8 +252,125 @@ def test_breadcrumb_doesnt_containt_series_url(self):
             html=True,
         )
 
-    @override_settings(IS_EXTERNAL_ENV=True)
-    def test_load_in_external_env(self):
-        """Test the page loads in external env."""
+    def test_pagination_is_not_shown(self):
         response = self.client.get(self.basic_page_url)
-        self.assertEqual(response.status_code, 200)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
```

Comment:
non-blocking suggestion: for simpler assertions, do it directly

```suggestion
        self.assertNotContains(response, 'class="ons-pagination__link"')
```

Reply:
Hi Dan 
Yes I agree that its simplified but it is not consistent with other tests in the code